rfc6962-bis.xml

```diff
@@ -1226,6 +1226,11 @@ entry specified by <spanx style="verb">start</spanx>.
           </t>
         </section>
       </section>
+      <section title="TLS Feature Extension">
+	<t>
+	  A CA MAY include the <xref target="tls_transinfo_extension">transparency_info</xref> extension number in a <xref target="RFC7633">TLS Feature Extension</xref> in root or intermediate certificates. This indicates that CT compliance is required for all certificates which are directly or indirectly issued by such certificates.
+	</t>
+      </section>
```

Comment:
OK, will do.  Do you have any preference on which option I should pick?

(Meanwhile, this PR LGTM).